manifests/d/DuckStudio/ChineseGit/2.5/DuckStudio.ChineseGit.locale.zh-CN.yaml

```
@@ -0,0 +1,31 @@
+# Created with YamlCreate.ps1 v2.4.1 $debug=NVS1.CRLF.5-1-22621-2506.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: DuckStudio.ChineseGit
+PackageVersion: "2.5"
+PackageLocale: zh-CN
+Publisher: DuckStudio 鸭鸭「カモ」
```

```suggestion
Publisher: DuckStudio
```
This should exactly match Control Panel